public/modules/core/css/core.css

```diff
@@ -194,7 +194,7 @@ div.home div.col-lg-12 {
 }
 
 div.col-lg-12.row-group {
-    margin-top: 35px;
+    margin-top: 0px;
 }
```

Comment:
Emi, this selector will affect everything that uses the col-lg-12 class, row-group, and is a div. This will have unintended consequences across the app. Add more specificity for the specific page of the app you are referring to.